.gitattributes

```diff
@@ -1,2 +1,5 @@
 * text=auto
 *.js text eol=lf
+/.github/ export-ignore
+/yarn.lock export-ignore
+/fievel-mousekewitz48.gif export-ignore
```

Comment:
Wouldn't that imply that no `.gif` files are to be used in the package's styling? If anything, I'd at least limit a general `*.gif` exclusion to the root directory only.